docs/host/quick-start.md

```diff
@@ -0,0 +1,60 @@
+
+# Hosting Quick Start
+
+###### On This Page: ######
```

Comment:
Remove this from all files. You'll notice on the right side of every page, this is automatically created by Docusaurus.